fs_mol/utils/metrics.py

```diff
@@ -70,7 +69,7 @@ def avg_metrics_over_tasks(
         # this returns, for each task, a dictionary of aggregated results
         aggregated_metrics[task] = avg_task_metrics_list(results)

```

Comment:
Looking at the types here, I'm a bit confused - `aggregated_metrics` should be `Dict[str, Dict[str, Tuple[float, float]]]` (as that is what comes back from `avg_task_metrics_list`), but then we are feeding this back into `avg_task_metrics_list` in at the end, which will then do something strange when it applies `np.mean` / `np.std` to the tuples it get. I think at some point we should maybe project the tuples we get back onto a single value representing the mean?

Reply:
+1; `np.mean` on `List[Tuple[float, float]]` seems to average out over _all_ the values, summing up means and std devs together, which doesn't sound great...

Reply:
sorry, I've messed up trying to fix another problem. Fixing

Reply:
Argh. Yes.